samples/basic/build.gradle

```diff
@@ -27,6 +27,7 @@ android {
 }
 
 dependencies {
+    implementation(project(':samples:lib-module'))
```

Comment:
Maybe in addition to that we could refer to the `Placeholder` class in the sample app code.

Reply:
I've checked final apk and `Placeholder` class is there (at least in `debug` variant) without referencing it explicitly from the code, but yes, let's add it in case if compiler becomes smarter.